model/src/lib.rs

```diff
@@ -728,6 +728,32 @@ impl FundingFee {
         })
     }
 
+    /// Calculate the fee paid or earned for a party in a particular
+    /// position.
+    ///
+    /// A positive sign means that the party in the `position` passed
+    /// as an argument is paying the funding fee; a negative sign
+    /// means that they are earning the funding fee.
+    fn compute_relative(&self, position: Position) -> SignedAmount {
```

Comment:
Would it make sense to reuse `FeeFlow` here for the return value?

Reply:
Very possibly! Would you be okay with taking over this PR and making that change if you think it would make things better? Otherwise we can continue with it next week.

Reply:
Yeah there is no rush, we can even merge it now and do it as a follow-up!

Reply:
I've been exploring this proposed change a little and I have some doubts.

At the moment this method is only used for logging. My idea was that it could be useful context to know who (role) pays what funding fee during a rollover. `FeeFlow` reflects fee payment between different positions, which is not quite the same. What do you think is more useful? I prefer the current one because it's easier to, for example, say: "the maker is earning 400 satoshi with this rollover which covers 4 hours".

Relatedly, I was considering implementing display on a dedicated type, in order to print something like `earns 400 satoshi` instead of `-400`. But it's just for a minor log, so probably not worth it.

Reply:
I'm gonna go ahead with this merge and we can later on decide if we want to make the change you suggested here.